jib-maven/pipeline.yaml

```diff
@@ -36,6 +36,8 @@ spec:
     params:
     - name: DIRECTORY
       value: $(params.SUBDIR)
+    - name: MAVEN_IMAGE
+      value: gcr.io/cloud-builders/mvn@sha256:57523fc43394d6d9d2414ee8d1c85ed7a13460cbb268c3cd16d28cfb3859e641
```

Comment:
Listing the image here will make it difficult to maintain, basically we would need to verify that that image very often